packages/auth/package.json

```diff
@@ -43,8 +43,5 @@
   "devDependencies": {
     "rimraf": "3.0.2",
     "vite": "2.9.15"
-  },
-  "engines": {
-    "node": ">=10"
   }
```

Comment:
`@coveo/auth` uses the fetch method from `browser-fetch.ts`, which only contains:
```ts
export type BrowserFetch = typeof fetch;

export function getBrowserFetch(): BrowserFetch {
  const isBrowser = typeof window !== 'undefined';
  return isBrowser ? fetch : () => Promise.resolve(new Response());
}
```

Given that it doesn't support Node.js (yet?), I'm not sure if we should or should not have `engines` here. Opinions?